com.belmoussaoui.snowglobe.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "com.belmoussaoui.snowglobe",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "command": "snowglobe",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=all",
+        "--socket=wayland",
+        "--own-name=org.qemu",
+        "--filesystem=home",
+        "--env=G_MESSAGES_DEBUG=all"
```

Comment:
why this verbosity at runtime? It will just flood the logs if anything.

Reply:
Copy paste failure, thanks